generate_gt_from_box.py

```diff
@@ -2,18 +2,20 @@
 
 import argparse
 import io
-import unicodedata
```

Comment:
Why do you propose to remove the import? `unicodedata` is used.

Reply:
Not in this script AFAICS

Reply:
Ah, sorry. My bad.